src/orca-jedi/state/State.cc

```diff
@@ -83,10 +83,12 @@ State::State(const Geometry & geom,
     readFieldsFromFile(nemo_file_name, *geom_, validTime(), "background",
          stateFields_);
     nemo_file_name = params.errorFieldFile.value().value_or("");
-    readFieldsFromFile(nemo_file_name, *geom_, validTime(), "background error standard deviation",
-       stateFields_);
-    readFieldsFromFile(nemo_file_name, *geom_, validTime(), "background error variance",
-       stateFields_);
+    if (nemo_file_name != "") {
+      readFieldsFromFile(nemo_file_name, *geom_, validTime(), "background error standard deviation",
+         stateFields_);
+      readFieldsFromFile(nemo_file_name, *geom_, validTime(), "background error variance",
+         stateFields_);
+    }
```

Comment:
```suggestion
    if (params.errorFieldFile.value()) {
      readFieldsFromFile(nemo_file_name, *geom_, validTime(), "background error standard deviation",
         stateFields_);
      readFieldsFromFile(nemo_file_name, *geom_, validTime(), "background error variance",
         stateFields_);
    }
```
Is maybe a little clearer still. Apologies, this bit is a little hacky on my part - we are still trying to firm up the distinction between error standard deviation and error variance in orca-jedi.